src/library/components/researcher-report-form/index.js

```diff
@@ -115,7 +115,7 @@ export default class ResearcherReportForm extends React.Component {
       }
     })(_fieldName)
 
-    if ((queryCache[cacheKey] != null ? queryCache[cacheKey].then : undefined)) { // already made a Promise that is still pending
+    if ((queryCache[cacheKey] !== null ? queryCache[cacheKey].then : undefined)) { // already made a Promise that is still pending
```

Comment:
`==` and `!=` are preferable for null comparisons because it allows null/undefined to be handled together. When I see `===`/`!==` comparisons with `null`/`undefined` I find myself wondering why the code is making that distinction and whether the other is being handled correctly. The ESLint `eqeqeq` rule can be configured to allow such null comparisons.

Reply:
@kswenson Thanks for noticing, and for the explanation.

Reply:
👍